docs/get-blocks.md

```diff
@@ -23,6 +23,7 @@ Block ID		2fb7571a6ccf02f3ac42f27c14ce0a4cb119060e4fbd7af36fd51894465e7002
 Prent ID		1c5a6267ba9512e141e4e90630cb326cecfbf6113818487449efeb37fc98ca18
 Timestamp		2021-03-19 17:46:15.973305066 +0000 UTC
 Height			12884163
+Status		        2 (Sealed)
```

Comment:
I would just show the words status not the number at all since it has no real meaning.